robot/rospackages/src/task_handler/scripts/task_handler_server.py

```diff
@@ -106,10 +161,48 @@ def handle_task(task, status, args):
     return response + "\n"
 
 def task_handler_server():
+    global local
```

Comment:
I think that running it locally should be an argument, also may allow us to call this function externally eventually  for testing using an argument

Reply:
Fixed, thanks